frontend/resources/translations/hin.json

```diff
@@ -0,0 +1,744 @@
+          "filePickerDialogTitle": "एक .flowy_plugin फ़ाइल चुनें",
+          "urlUploadFailure": "URL खोलने में विफल: {}"
+        },
+        "theme": "थीम",
+        "builtInsLabel": "डिफ़ॉल्ट थीम",
+        "pluginsLabel": "प्लगइन्स",
+        "showNamingDialogWhenCreatingPage": "पेज बनाते समय नामकरण संवाद दिखाएँ"
+      },
+      "files": {
+        "copy": "कॉपी करें",
+        "defaultLocation": "फ़ाइलें और डेटा संग्रहण स्थान पढ़ें",
+        "exportData": "अपना डेटा निर्यात करें",
+        "doubleTapToCopy": "पथ को कॉपी करने के लिए दो बार टैप करें",
+        "restoreLocation": "AppFlowy डिफ़ॉल्ट पथ पर रीस्टार्ट करें",
+        "customizeLocation": "कोई अन्य फ़ोल्डर खोलें",
+        "restartApp": "परिवर्तनों को प्रभावी बनाने के लिए कृपया ऐप को रीस्टार्ट करें।",
+        "exportDatabase": "डेटाबेस निर्यात करें",
+        "selectFiles": "उन फ़ाइलों का चयन करें जिन्हें निर्यात करने की आवश्यकता है",
+        "selectAll": "सभी का चयन करें",
+        "deselectAll": "सभी को अचयनित करें",
+        "createNewFolder": "एक नया फ़ोल्डर बनाएँ",
+        "createNewFolderDesc": "हमें बताएं कि आप अपना डेटा कहां संग्रहीत करना चाहते हैं",
+        "defineWhereYourDataIsStored": "परिभाषित करें कि आपका डेटा कहाँ संग्रहीत है",
+        "open": "खुला",
+        "openFolder": "मौजूदा फ़ोल्डर खोलें",
+        "openFolderDesc": "इसे पढ़ें और इसे अपने मौजूदा AppFlowy फ़ोल्डर में लिखें",
+        "folderHintText": "फ़ोल्डर का नाम",
+        "location": "एक नया फ़ोल्डर बनाना",
+        "locationDesc": "अपने AppFlowy डेटा फ़ोल्डर के लिए एक नाम चुनें",
+        "browser": "ब्राउज़ करें",
+        "create": "बनाएँ",
+        "set": "सेट",
+        "folderPath": "आपके फ़ोल्डर को संग्रहीत करने का पथ",
+        "locationCannotBeEmpty": "पथ खाली नहीं हो सकता",
+        "pathCopiedSnackbar": "फ़ाइल संग्रहण पथ क्लिपबोर्ड पर कॉपी किया गया!",
+        "changeLocationTooltips": "डेटा निर्देशिका बदलें",
+        "change": "परिवर्तन",
+        "openLocationTooltips": "अन्य डेटा निर्देशिका खोलें",
+        "openCurrentDataFolder": "वर्तमान डेटा निर्देशिका खोलें",
+        "recoverLocationTooltips": "AppFlowy की डिफ़ॉल्ट डेटा निर्देशिका पर रीसेट करें",
+        "exportFileSuccess": "फ़ाइल सफलतापूर्वक निर्यात करें!",
+        "exportFileFail": "निर्यात फ़ाइल विफल!",
+        "export": "निर्यात"
+      },
+      "user": {
+        "name": "नाम",
+        "email": "ईमेल",
+        "tooltipSelectIcon": "आइकन चुनें",
+        "selectAnIcon": "एक आइकन चुनें",
+        "pleaseInputYourOpenAIKey": "कृपया अपनी OpenAI key इनपुट करें",
+        "clickToLogout": "वर्तमान उपयोगकर्ता को लॉगआउट करने के लिए क्लिक करें"
+      },
+      "shortcuts": {
+        "shortcutsLabel": "शॉर्टकट",
+        "command": "कमांड",
+        "keyBinding": "कीबाइंडिंग",
+        "addNewCommand": "नया कमांड जोड़ें",
+        "updateShortcutStep": "इच्छित key संयोजन दबाएँ और ENTER दबाएँ",
+        "shortcutIsAlreadyUsed": "यह शॉर्टकट पहले से ही इसके लिए उपयोग किया जा चुका है: {संघर्ष}",
```

Comment:
replace {संघर्ष} with {conflict}

Wherever you see something enclosed within {} (curly braces) that is actually a variable name, don't translate them. They act as a placeholder.